spec/features/idv/doc_auth/idv_flow_spec.rb

```diff
@@ -0,0 +1,276 @@
+        expect(profile.active).to be(true)
+        expect(profile.decrypt_pii(user.password)).to have_attributes(
+          first_name: 'LICENSE',
+          middle_name: nil,
+          last_name: 'SAMPLE',
+          dob: '1966-05-05',
+          ssn: Idp::Constants::MOCK_IDV_APPLICANT_WITH_SSN[:ssn],
+          phone: phone_number,
+          address1: '123 ABC AVE',
+          address2: 'APT 3E',
+          city: 'ANYTOWN',
+          state: 'NY',
+          zipcode: '12345',
+          same_address_as_id: nil,
+          identity_doc_address1: nil,
+          identity_doc_address2: nil,
+          identity_doc_city: nil,
+          identity_doc_zipcode: nil,
+          identity_doc_address_state: nil,
+          state_id_jurisdiction: 'NY',
+          issuing_country_code: nil,
+        )
+      end
+    end
+
+    describe 'IDV Passport Flow' do
+      let(:address1) { '123 Main St' }
+      let(:city) { 'Anywhere' }
+      let(:state_abbr) { 'VA' }
+      let(:state) { 'Virginia' }
+      let(:zipcode) { '66044' }
+      let(:fake_dos_api_endpoint) { 'http://fake_dos_api_endpoint/' }
+      let(:doc_auth_response) { LexisNexisFixtures.true_id_response_passport_without_tamper }
+
+      before do
+      end
+
+      scenario 'When the user completes all IDV steps' do
+        visit_idp_from_oidc_sp_with_ial2
+        sign_in_and_2fa_user(user)
+        visit idv_welcome_path
+        complete_welcome_step
+        complete_agreement_step
+        complete_hybrid_handoff_step
+        complete_choose_id_type_step(choose_id_type: 'passport')
+        attach_passport_image(
+          Rails.root.join('spec', 'fixtures', 'doc_auth_images', 'passport.jpg'),
+        )
+        submit_images
+        complete_ssn_step
+        fill_in 'idv_form_address1', with: address1
+        fill_in 'idv_form_city', with: city
+        select state, from: 'idv_form_state'
+        fill_in 'idv_form_zipcode', with: zipcode
+        click_idv_continue
+        complete_verify_step
+        fill_out_phone_form_ok(phone_number)
+        verify_phone_otp
+        complete_enter_password_step(user)
+
+        expect(page).to have_current_path(idv_personal_key_path)
+
+        profile = user.profiles.last
+
+        expect(profile.active).to be(true)
+        expect(profile.decrypt_pii(user.password)).to have_attributes(
+          first_name: 'DAVID',
+          middle_name: 'PASSPORT',
+          last_name: 'SAMPLE',
+          dob: '1986-07-01',
+          ssn: Idp::Constants::MOCK_IDV_APPLICANT_WITH_SSN[:ssn],
+          phone: phone_number,
+          address1:,
+          address2: '',
+          city:,
+          state: state_abbr,
+          zipcode:,
+          same_address_as_id: nil,
+          identity_doc_address1: nil,
+          identity_doc_address2: nil,
+          identity_doc_city: nil,
+          identity_doc_zipcode: nil,
+          identity_doc_address_state: nil,
+          state_id_jurisdiction: nil,
+          issuing_country_code: 'USA',
```

Comment:
This may be a result of the fixture (and thus in practice out of our control), but the Jira ticket specified the value should be either 'US' or 'Department of State'.

Does it matter?